docs/api/migration-guides/v2-primitives.mdx

```diff
@@ -349,7 +348,8 @@ sampler.set_options(shots=2000, optimization_level=0)
 ### Error mitigation and suppression
 
 * Because Sampler V2 returns samples without postprocessing, it does not support resilience levels.
-* Sampler V2 does not currently support `optimization_level`.
+* Sampler V2 does not support `optimization_level`.
+* In `qiskit_ibm_runtime` 0.25 and later, `optimization_level` in Estimator V2 is deprecated.
```

Comment:
It's also deprecated on the server side, so using <0.25 qiskit-ibm-runtime will not do you any good after the deprecation period. 

```suggestion
* Estimator V2 will drop support for `optimization_level` on or around 30 September 2024. 
```